### ISSUE TITLE
Remove ScoringMode - only one mode is ever used. 

### ISSUE THREAD

**Issue by julesjacobsen** [exomiser/Exomiser#149]
I can't remember which Prioritiser used ScoringMode.RANK_BASED  but none of the current ones do. They all use ScoringMode.RAW_SCORE.

Using this triggers the RankBasedGeneScorer which runs the RawScoreGeneScorer and then alters the scores afterwards.

I think given this is unused it should be removed. This functionality can be implemented in the Prioritiser directly when producing the PrioritiserScores. As it is it just adds unnecessary complexity.

Any comments in favor of retaining this?

**Comment by drseb** [comment]
I was wondering about this as well. Please go ahead an get rid of it.

**Comment by julesjacobsen** [comment]
Cool. I'd like to remove the ExomeWalker and Uberpheno Prioritisers too. ExomeWalker can be run using HiPhive using ppi only mode and Uberpheno is not actually available for people to run. That OK with you?

Also what's up with Phenix2? Can you upgrade the existing Phenix with the new code?

**Comment by drseb** [comment]
> Cool. @drseb I'd like to remove the ExomeWalker and Uberpheno Prioritisers too. ExomeWalker can be run using HiPhive using ppi only mode and Uberpheno is not actually available for people to run. That OK with you?

Sure. I suggest to speak with @pnrobinson as well, if not done already

> Also what's up with Phenix2? Can you upgrade the existing Phenix with the new code?

I made a separate branch. Not sure this can be merged easily. Before releasing phenix2, I have to make way more simulation tests etc. Using BOQA as it has been published will not work very well for the use-case of variant(gene)-prioritization. Once this is done, we need at least one instance of exomiser-web, with only phenix2 as option on a tomcat at charite. I hope this will work.

**Comment by pnrobinson** [comment]
I am OK with merging ExomeWalker into HiPHIVE

I would like to run one flavour of ExomeWalker here at JAX, we should talk about how package everything by skype maybe next week


Peter Robinson

Professor of Computational Biology

The Jackson Laboratory for Genomic Medicine

10 Discovery Drive

Farmington, CT 06032

860.837.2095 t | 860.990.3130 m

peter.robinson@jax.org<mailto:peter.robinson@jax.org>

www.jax.org



The Jackson Laboratory: Leading the search for tomorrow's cures


________________________________

**Comment by damiansm** [comment]
I think exomewalker is different from using hiphive with ppi only so i
would not remove it without a lot of refactoring?

**Comment by pnrobinson** [comment]
If it is hard to refactor, then I can keep a separate version of Exome Walker here -- just need to add a redirect at the Charite for a while. I think there are use cases where people will want ExomeWalker rather than HiPHIVE. An issue is that I am using my old Exomiser code -- it would be great to refactor to Jules' code base. After this grant today I would love to work on this -- maybe we can skype early next week to make a game plan?


Peter Robinson

Professor of Computational Biology

The Jackson Laboratory for Genomic Medicine

10 Discovery Drive

Farmington, CT 06032

860.837.2095 t | 860.990.3130 m

peter.robinson@jax.org<mailto:peter.robinson@jax.org>

www.jax.org



The Jackson Laboratory: Leading the search for tomorrow's cures


________________________________

**Comment by julesjacobsen** [comment]
OK - I'll keep ExomeWalker as is, but will remove Uberpheno.

@pnrobinson sure, name a date and time. 

Will close this and migrate to new issue as the original issue has been done!
